### PR TITLE
fix uwebsocket import and websocket handshake

### PR DESCRIPTION
**Pull request by levsh**
- Fixed builtin uwebsocket module import.
- Fixed server handshake. Add extra '\r\n' after last header line to indicate end of the header. Without it websocket server doesn't work properly with safari browser on mac.

### CODE REVIEW

File: uasyncio.websocket.server/uasyncio/websocket/server.py [hunk 1]
```diff
@@ -50,10 +49,9 @@ def WSReader(reader, writer):
 Connection: Upgrade\r
 Sec-WebSocket-Accept: """)
         await writer.awrite(respkey)
-        # This will lead to "<key>\n\r\n" being written. Not exactly
```
Comment (pfalcon) [inline uasyncio.websocket.server/uasyncio/websocket/server.py]: It's unclear why this change is made, not mentioned in commit message.

**Comment by pfalcon** [comment]
> i split PR into two commits  and revert one change.

Thanks! So, why "Add extra '\r\n' after the last server header line to indicate the end of the header" change is needed. As the comment there says, previously "\n\r\n" was sent at the end (missing first "\r"), and it worked with common browsers like Firefox and Chrome. Can you elaborate where it didn't work?

Also, any reason why you used .rstrip(), while there already was `#[:-1]` comment there? 

(Sorry, this may seem as nitpicking, but it's actually all optimizations for less CPU cycles and memory allocation. I'd like to be efficient with both on the level of system libs, to let user apps do what they want (instead of running out of memory)).

**Comment by levsh** [comment]
@pfalcon Firefox and Chrome both work on master branch for my test code. But Safari doesn't. I used mozilla [documentation](https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API/Writing_WebSocket_servers#Server_handshake_response) and wireshark to find difference between uasyncio and MicroWebSrv websocket implementation(my code works good with MicroWebSrv and Safari). As i understand without `\r\n` Safari doesn't establish websocket connection properly and send or received any data.
And you are right `#[:-1]` faster than `rstrip` method. I will fix it.

**Comment by pfalcon** [comment]
> As i understand without \r\n Safari doesn't establish websocket connection properly and send or receive any data.

Ack, thanks for explanation!

> And you are right #[:-1] faster than rstrip method. I will fix it.

If you have time. Otherwise, feel free to leave it for me, I'm happy to do minor cleanups, especially for first-time contributors ;-).

**Comment by levsh** [comment]
@pfalcon done

**Comment by pfalcon** [comment]
Thanks for the cleanups!

I still expanded commit messages to describe the matters in more detail, helps when maintaining a hundred of modules over years.

Merged and released as 0.2.

**Comment by pfalcon** [comment]
Oh, and btw, my understanding is that need for "Server:" header for Safari was initial hypothesis. But turned out, the actual issue was borked line-endings in existing headers, that's why "Server:" patch is gone.
